x-pack/solutions/security/plugins/security_solution/common/detection_engine/constants.ts

```diff
@@ -60,3 +60,15 @@ export const SUPPRESSIBLE_ALERT_RULES_GA: Type[] = [
   'threat_match',
   'machine_learning',
 ];
+
+export const RISK_SCORE_LOW = 21;
+export const RISK_SCORE_MEDIUM = 47;
+export const RISK_SCORE_HIGH = 73;
+export const RISK_SCORE_CRITICAL = 99;
```

Comment:
this is very subjective, but I would add a comment here to explain what these are. Are these values out of 100?

Reply:
@PhilippeOberti 
Yes, this is out of 100. 

 I have also included a [link to documentation](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#rule-ui-basic-params) ( See point _e._ ) regarding this in PR description. See if that makes sense.

Reply:
it does, I'm just thinking of developers who are not super familiar with this looking at the code... It's all good, it was a subjective comment, feel free to ignore!

Reply:
Yes, makes sense. I will add a TSDoc with Elastic documentation. Thanks for pointing it out.